MANIFEST.in

```diff
@@ -0,0 +1,37 @@
+recursive-include unionml/templates *
+
+include README.md
+include src/pythae/py.typed  # marker file for PEP 561
+
+include CHANGELOG.md
+include changelog.md
+include LICENSE
+include LICENSE.txt
+include CITATION.cff
+include *.cff  # citation info
+
+include MANIFEST.in
+include src/pythae/pyproject.toml
+include setup.py
+include setup.cfg
+include tox.ini
+
+include requirements.txt
+
+
+recursive-exclude tests *
+recursive-exclude docs *
+recursive-exclude examples *
+recursive-exclude site *
+recursive-exclude .github *
+recursive-exclude .archives *
+
+exclude .gitignore
+exclude .gitattributes
+exclude .pre-commit-config.yaml
+exclude .readthedocs.yml
+exclude codecov.yml
+
+exclude Dockerfile
+exclude Makefile
```

Comment:
No such file in repo

Reply:
As I stated earlier, this is mostly from a template MANIFEST.in that I use, Makefile is often common in a repository. 

Especially if you plan on collaborative development, with a Makefile it gets lot easier to run linters, formatters, isort, pytest (test coverage), type checking, docstrings coverage, etc. 

I would recommend you adding a Makefile and using flake8, black, isort, mypy, interrogate for maintaining listing, formatting, etc.

Reply:
Thank you for the suggestion, I will definitely consider adding a Makefile in the near future so let's keep the `exclude Makefile` and remove ` exclude Dockerfile`

Reply:
Done